furnace.c

```diff
@@ -215,7 +235,9 @@ command_handler(furnace_context_t* ctx, uint8_t* buffer, void (*feedback)(const
       const size_t msg_len = sizeof(msg)-1;
       feedback(msg, msg_len);
     }
-  } else if (strncmp(buffer, "auto\n", 5) == 0) {
+  } 
+#if CONFIG_AUTO == CONFIG_AUTO_PILOT
+  else if (strncmp(buffer, "auto\n", 5) == 0) {
```

Comment:
`strcmp()` is fine against constant strings (instead of `strncmp()` - we don't have to calculate the length of it manually.

Reply:
Actually, whole command_handler is written using `strncmp()`, i was just using the same convention.